flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackendConfigTest.java

```diff
@@ -97,6 +97,18 @@ public void testDefaultsInSync() throws Exception {
         assertEquals(defaultIncremental, backend.isIncrementalCheckpointsEnabled());
     }
 
+    @Test
+    public void testDefaultDbLogDir() throws Exception {
+        final EmbeddedRocksDBStateBackend backend = new EmbeddedRocksDBStateBackend();
+        final File logFile = File.createTempFile(getClass().getSimpleName() + "-", ".log");
+        // set the environment variable 'log.file' with the Flink log file location
+        System.setProperty("log.file", logFile.getPath());
+        assertEquals(
+                logFile.getParent(),
+                backend.createOptionsAndResourceContainer().getDbOptions().dbLogDir());
```

Comment:
The created resouce container is native objects and we must ensure to close the `RocksDBResourceContainer` in the end of the test to avoid unexpected exception.